### PR TITLE
Update 12.84.2

### PR DESCRIPTION
**Pull request by oufmilo**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by oufmilo** [comment]
https://yunorunner.travelpandas.eu/ci/job/12

File: check_process [hunk 1]
```diff
@@ -10,15 +10,15 @@
 		setup_private=1
 		setup_public=1
 		upgrade=1
-		#12.83.0~ynh1
-		upgrade=1	from_commit=c8517f3f2ac290a25eff6e4c20e829830cb0658f
+		# 12.84.0~ynh1
+		#upgrade=1	from_commit=32c061ef1a6cedb00300ffd38184cd7d5fff4c95
```
Comment (ericgaspar) [inline check_process:14]:
```suggestion
		upgrade=1	from_commit=32c061ef1a6cedb00300ffd38184cd7d5fff4c95
```

**Review by ericgaspar — COMMENTED** [review]
Shall we test from the previous version?

**Comment by oufmilo** [comment]
!testme

**Pull request merged**